addons/folo/model-java/pom.xml

```diff
@@ -32,8 +32,12 @@
     	<artifactId>indy-model-core-java</artifactId>
     </dependency>
     <dependency>
-    	<groupId>junit</groupId>
-    	<artifactId>junit</artifactId>
+      <groupId>org.infinispan</groupId>
```

Comment:
What does this dependency do? It seems like something we might not want our client API users to have to deal with if we can avoid it.